helpers/strings/includes.js

```diff
@@ -3,20 +3,22 @@
  * check if a substring exist within a string. This is very similiar to the
  * indexOf helper, except it uses String.prototype.includes() and returns a
  * boolean.
+ * note: handlebars returns booleans as strings, so only return a value if the substring is found
+ * otherwise, return undefined (rather than false)
  *
  * @param  {string} string
  * @param  {string} substring
- * @return {boolean}
+ * @throws {Error} if either arg isn't a string
+ * @return {undefined|boolean}
  */
 module.exports = function (string, substring) {
-  if (!string || typeof string !== 'string' || typeof substring !== 'string') {
+  if (!string || !substring) {
+    return; // string may simply not be defined
```

Comment:
its okay this returns `undefined` if `substring` or `string` are undefined right?

Reply:
yes, that's what it's doing

Reply:
👍 Just making sure that was the intention vs `typeof(string) === "undefined" || typeof(substring) === "undefined"`